tests/python/pytorch/graphbolt/test_graphbolt_utils.py

```diff
@@ -3,6 +3,50 @@
 import torch
 
 
+def test_unique_and_compact_node_list_hetero():
```

Comment:
Update unit test?

Reply:
This is just the test for new added API.